### ISSUE TITLE
update.sh is saved in DOS format instead of Unix format

### ISSUE THREAD

**Issue by Forceu** [grocy/grocy#412]
Every time I am updating my Grocy instance with the update.sh, I need to convert it to the unix format first. The script is unable to run, as it interprets the \r line endings as a character.

[Bash output](https://pastebin.com/0piHvq6H)

This can be fixed if I run `dos2unix update.sh` before running the update.sh script. However, with every update the update script is overwritten.

As far as I am aware, this seems only to be an issue with the update.sh in the release archive.

**Comment by Forceu** [comment]
Might be related to this issue: https://stackoverflow.com/a/40821931/819367

**Comment by berrnd** [comment]
Thanks for the hint, also already searched for this and found something similar (that will be the problem, as I only use Windows when working on grocy...).

**Comment by makcuber** [comment]
I am getting this error for every file I open in the Grocy archive after attempting to install it. my apache vhost is setup correctly so far as I can tell, but it only shows a blank page. I tested it by adding a plain html index file to the webroot and it loaded just fine, so something about the index.php is not working and I suspect its because its in dos format and not unix. Im using the latest release version

**Comment by berrnd** [comment]
PHP can handle both line endings. Blank page errors were mostly about wrong rewriting rules, wrong root directory settings or something similar in the past - please check the webserver and PHP logs...

**Comment by makcuber** [comment]
I was able to get it working by cloning the repo directly, which eliminated the DOS format issue. Once I got composer, yarn, directory permissions, and the apache vhost all setup correctly it worked like a charm. I will say though that there is a serious lack of documentation for getting it set up, but its manageable if you are familiar with modern PHP development or just invest a good chunk of time googling issues.
